src/open_inwoner/openzaak/admin.py

```diff
@@ -90,6 +123,85 @@ class CatalogusConfigAdmin(admin.ModelAdmin):
     ordering = ("domein", "rsin")
     list_filter = ("service",)
 
+    actions = ["export_catalogus_configs"]
+
+    def get_urls(self):
+        urls = super().get_urls()
+        custom_urls = [
+            path(
+                "import-catalogus-dump/",
+                self.admin_site.admin_view(self.process_file_view),
+                name="upload_zgw_import_file",
+            ),
+        ]
+        return custom_urls + urls
+
+    @admin.action(description=_("Exporteer naar json export"))
```

Comment:
```suggestion
    @admin.action(description=_("Exporteer naar json"))
```
2 x "export" is one too many?